sdk/formrecognizer/Azure.AI.FormRecognizer/src/BoundingPolygon.cs

```diff
@@ -56,6 +56,11 @@ internal BoundingPolygon(IReadOnlyList<PointF> points)
         /// <returns>A <see cref="PointF"/> corresponding to the specified <paramref name="index"/>.</returns>
         public PointF this[int index] => Points[index];
 
+        /// <summary>
+        /// Gets the number of points of this <see cref="BoundingPolygon"/>.
+        /// </summary>
+        public int Length => Points.Length;
```

Comment:
Part of https://github.com/Azure/azure-sdk-for-net/issues/29398. I'm not closing the issue yet because we still need to update samples to use this property. Since updating the samples now could confuse costumers using the latest preview, I'll wait until it's closer to our next release to update them.